pxtblocks/code-validation/validateBlocksInSetExist.ts

```diff
@@ -0,0 +1,20 @@
+namespace pxt.blocks {
+    // validates that a combination of blocks in the set satisfies the required count
+    // returns the blocks that make the validator pass
+    export function validateBlocksInSetExist({ usedBlocks, blocks, count }: {
+        usedBlocks: Blockly.Block[],
+        blocks: string[],
```

Comment:
nit: perhaps a slightly more descriptive name could be helpful here, i.e. 'blockIdsToCheck'

Reply:
Quick question, if I change the name here, should I update the name in the validator plans and the interface definition as well? I know changing the parameter in the function is a bit more pertinent since that's where we'd interface with the validator elsewhere, and having the name be more descriptive is helpful when using the validator. I'm wondering if there should be consistency underneath as well.

Reply:
I'd aim for consistency wherever possible :)

Reply:
Actually I take it back in this case. I think blockIdsToCheck is too wordy for the actual validator plan. For that, I'd go with blocks (as you have it) or blockIds. The "toCheck" is a little less necessary, since there's no notion of the "usedBlocks" in that scenario, which is part of what causes the possible-confusion in this function.